src/main/java/com/aws/greengrass/util/platforms/StubResourceController.java

```diff
@@ -20,12 +23,12 @@ public void removeResourceController(GreengrassService component) {
 
     @Override
     public void updateResourceLimits(GreengrassService component, Map<String, Object> resourceLimit) {
-        // no op
+        LOGGER.warn("System resource limits for components not supported on this platform.");
```

Comment:
nit: you don't need the "." the logger will append one

Reply:
GTK. Thanks.